services/movie_session.py

```diff
@@ -42,3 +42,12 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
Consider adding exception handling for `MovieSession.DoesNotExist` in the `delete_movie_session_by_id` function to manage cases where a session with the given ID does not exist. This will prevent the application from crashing and allow you to handle the error gracefully.